packages/cli/src/command.js

```diff
@@ -43,6 +49,19 @@ export default class NuxtCommand {
     }
 
     return Promise.resolve(this.cmd.run(this))
+      .then(() => {
+        if (this.forceExit) {
```

Comment:
What's your idea to move this delayed util to the utils?

Reply:
My main reason for not doing that was because you dont always want to forceExit so you'd still need that if-statement. And I didnt see much re-usability for the setTimeout part, but can change it anyway. Will then also change the order, so the .then is only added when we want to set the forceExit timeout.

Reply:
Can be reused by either programmatic users or workers even if kept in cli package but refactored into a util :) Anyway just a comment.